aws-lambda-java-tests/src/test/resources/apigw/events/apigw_event2.json

```diff
@@ -0,0 +1,62 @@
+{
```

Comment:
Instead of `2` as a suffix for JSON event files, could we put in more descriptive name (such as, for example, `apigw_event_null_headers.json`)?

Reply:
sure, will do